packages/amplify-graphql-api-construct-tests/src/__tests__/sql-models.test.ts

```diff
@@ -10,11 +10,16 @@ import {
   storeDbConnectionConfig,
   storeDbConnectionConfigWithSecretsManager,
   deleteDbConnectionConfigWithSecretsManager,
+  deleteDbConnectionStringConfig,
```

Comment:
Can you add a test case for db connection string?

Reply:
I've added a unit test to ensure that the backend changes like SQL Lambda env variables, SSM permissions etc are as expected. In order for an E2E to successfully run, I would need the corresponding changes released in Lambda Layer. I have created a separate task to add an E2E as a follow-up.

Reply:
The lambda layer changes are in beta and available for E2E testing

Reply:
The CDK construct tests do not use the Beta lambda layer, only the Gen-1 E2Es use it.